### ISSUE TITLE
wite text on solid background

### ISSUE THREAD

**Issue by crspl** [flameshot-org/flameshot#2358]
### Feature Description

now it's possible to add text but only on transparent background so only letters have color. It makes is hard to read depends on what's below. Idea is to add possibility to set background color (default to transparent as it is now) so whole rectangle containing text would be in given color

![2022-02-03_12-42](https://user-images.githubusercontent.com/1543827/152336382-26373857-0461-4d8c-9cab-8fc37d8a698a.png)

**Comment by mmahmoudian** [comment]
I think this is related to #1249 which is aiming at improving the readability of text. I don't think we will add text with solid background (at least for now) as it is possible with combination of current tools. This has even made easier by a recent PR #2108.

**Comment by veracioux** [comment]
@crspl I agree with @mmahmoudian. There is no real benefit to this. You can just do the following:

https://user-images.githubusercontent.com/29044423/164979615-ca2e845a-96bb-40a5-8396-f8f4d0eb63c9.mp4

:

**Comment by MateuszJakubowski** [comment]
> @crspl I agree with @mmahmoudian. There is no real benefit to this. You can just do the following:
> 
>  Peek.2022-04-24.15-47.mp4 
> :

This is not true. If you need ad 20 texts in one screenshot, the benefit is high (you have to add 20 objects instead of 40). Especially when there is a need to move some text object after placement.

**Comment by JennytheDragon2000** [comment]
I also wish if flameshot had this functionality..
